main.go

```diff
@@ -1027,7 +1194,7 @@ func distBetweenAngles(endDeg, startDeg, originalDist float64) float64 {
 }
 
 // timeEst must be absolute value
-func sampleLinearVel(odometry movementsensor.MovementSensor, goalVel, timeEst float64) float64 {
+func sampleLinearVel(odometry movementsensor.MovementSensor, goalVel, timeEst float64, data *os.File, testType string, startPos float64) float64 {
 	maxSpeed := -0.000000001
 	start := time.Now()
 	for {
```

Comment:
alittle worried that we should have a sleep or ticker in this loop so we don't spam calls to the component. 

probably fine as is just calling out

Reply:
also a little worried about this, but I was trying to think about how to have it polling at the same frequency as the sensor and maybe that's a good option? I think this is at least alright for now